lib/gui/lvgl/lv_conf.h

```diff
@@ -46,6 +51,12 @@
 		CONFIG_LVGL_CUSTOM_CHROMA_KEY_BLUE)
 #endif
 
+#if CONFIG_LVGL_IMG_INDEXED_CHROMA
```

Comment:
Is there a reason this isn't `#ifdef` like everything else?

Reply:
nice catch, fixed